css/at-rules/page.json

```
@@ -44,6 +44,45 @@
             "deprecated": false
           }
         },
+        "page": {
+          "__compat": {
+            "description": "<code>page</code> property",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/page",
+            "spec_url": "https://w3c.github.io/csswg-drafts/css-page/#using-named-pages",
+            "support": {
+              "chrome": {
+                "version_added": "85"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "110"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": {
+                "version_added": "1.5"
+              },
+              "webview_android": {
+                "version_added": "37"
+              }
```

ChromeStatus says this was implemented in WebView 85, and I'd be surprised that Samsung Internet supported this so long ago.

```suggestion
              "samsunginternet_android": "mirror",
              "webview_android": "mirror"
```